src/app/shared/filter-bar/filter-bar.component.html

```diff
@@ -70,7 +70,7 @@
           <mat-select-trigger *ngIf="this.milestoneService.hasNoMilestones">
             <span>No Milestones</span>
           </mat-select-trigger>
-          <mat-option *ngFor="let milestone of this.milestoneService.milestones" [value]="milestone.title">
+          <mat-option *ngFor="let milestone of getDisplayedMilestones()" [value]="milestone.title">
```

Comment:
Not really sure if creating a method to obtain the available list of milestones is really necessary. I think we can make use of `ngIf` directive to conditionally render the default milestone options? See `filter-bar.component.html`, lines 27-31 to see how the directive is used to conditionally render the status checkboxes.

Reply:
Additionally, this also allows us to differ the text rendered from the value, i.e. the texts rendered should be `Issues without a milestone` and `PRs without a milestone`, the nouns with an `s`.